tests/unit/test_lifecycle.py

```diff
@@ -1310,6 +1310,65 @@ def test_prime_with_invalid_image_info_raises_exception(self):
                 lifecycle.execute, 'prime', self.project_options)
         self.assertThat(raised.image_info, Equals('not-json'))
 
+    def test_prime_with_launchpad_build_info_records_manifest(self):
+        self.useFixture(fixtures.EnvironmentVariable(
+            'SNAPCRAFT_BUILD_INFO', '1'))
+        test_launchpad_build_info = '{"build-url": "test-build-url"}'
```

Comment:
This is the only hint in this PR of what the build information dict might contain.  Could we have a bit of documentation with conventional values, or would you rather leave it entirely up to the build environment?